### PR TITLE
Fixed some problems on OMX/RaspberryPi

### PR DESCRIPTION
**Pull request by strfry**
I discovered some compile erros while trying stuff again back on the Raspi.
Also i fixed a problem with VP 8codec, related to YUV padding the old conversion code.

### CODE REVIEW

File: modules/omx/module.c [hunk 1]
```diff
@@ -87,7 +87,7 @@ int omx_vidisp_display(struct vidisp_st *st, const char *title,
 			frame->linesize[0], frame->linesize[1],
 			frame->linesize[2]);
 		err = omx_display_enable(st->omx,
-			frame->size.w, frame->size.h, frame->linesize[0]);
+			frame->size.w, frame->size.h, frame->size.w); //frame->linesize[0]);
```
Comment (alfredh) [inline modules/omx/module.c]: please delete commented out code.

File: modules/omx/omx.c [hunk 2]
```diff
@@ -251,28 +251,37 @@ int omx_display_enable(struct omx_state* st,
...
+		error("omx_display_enable: could not retrieve port definition\n");
+		err = ENOMEM;
+		goto exit;
+	}
+
+	warning("omx port def: h=%d w=%d s=%d sh=$d\n",
+		portdef.format.video.nFrameWidth,
+		portdef.format.video.nFrameHeight,
+		portdef.format.video.nStride,
+		portdef.format.video.nSliceHeight);
```
Comment (alfredh) [inline modules/omx/omx.c]: perhaps info-level is better ?

**Comment by strfry** [comment]
Fixed by your comments

File: modules/omx/omx.c [hunk 3]
```diff
@@ -251,28 +251,37 @@ int omx_display_enable(struct omx_state* st,
...
+		error("omx_display_enable: couldn't retrieve port def\n");
+		err = ENOMEM;
+		goto exit;
+	}
+
+	info("omx port definition: h=%d w=%d s=%d sh=$d\n",
+		portdef.format.video.nFrameWidth,
+		portdef.format.video.nFrameHeight,
+		portdef.format.video.nStride,
+		portdef.format.video.nSliceHeight);
```
Comment (alfredh) [inline modules/omx/omx.c]: the forth format specified says `$d` is that on purpose?
should it perhaps be `%d` instead?

**Pull request merged**